### PR TITLE
`TorchVisionBackbone` refactor

### PR DESCRIPTION
**Pull request by dnth**
This PR refactors the `TorchVisionBackbone` class. I've verified all models that uses the `TorchvisionBackbone` (UNet, Faster RCNN, Mask RCNN, Keypoint RCNN and Retinanet) to be working.

I tested out the torchvision retinanet in this notebook and now it is has better performance than before.

https://colab.research.google.com/gist/dnth/26275bdf9d79498046076618b8ff4a02/getting_started_object_detection.ipynb

### CODE REVIEW

**Comment by dnth** [comment]
Here is the training epochs using the refactored Torchvision Retinanet

image

File: icevision/models/torchvision/fastai_callbacks.py [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 from icevision.imports import *
 from icevision.engines.fastai import *
-from icevision.models.torchvision import faster_rcnn
+
+# from icevision.models.torchvision import faster_rcnn
```
Comment (fstroth) [inline icevision/models/torchvision/fastai_callbacks.py:6]: Will we need this line at a later point or can we just delete it?

**Review by fstroth — APPROVED** [review]
Looks good to me. Great work @dnth

File: icevision/models/torchvision/fastai_callbacks.py (hunk 1 above)
Comment (dnth) [reply icevision/models/torchvision/fastai_callbacks.py:6]: Ahh good catch.. I will just remove it..

**Pull request merged**